pandas/io/json.py

```
@@ -792,7 +792,10 @@ def _recursive_extract(data, path, seen_meta, level=0):
                     if level + 1 > len(val):
                         meta_val = seen_meta[key]
                     else:
-                        meta_val = _pull_field(obj, val[level:])
+                        try:
+                            meta_val = _pull_field(obj, val[level:])
+                        except:
```

don't use a bare except, list a specific exception ``KeyError``?